tests/schemas/avro.py

```diff
@@ -0,0 +1,401 @@
+from avro.schema import ArraySchema, Field, MapSchema, RecordSchema, UnionSchema
+from karapace.avro_compatibility import parse_avro_schema_definition
+
+import json
+
+schema1 = parse_avro_schema_definition(
+    json.dumps(
+        {
+            "type": "record",
+            "name": "myrecord",
+            "fields": [
+                {"type": "string", "name": "f1"},
+            ],
+        },
+    )
+)
+schema2 = parse_avro_schema_definition(
+    json.dumps(
+        {
+            "type": "record",
+            "name": "myrecord",
+            "fields": [
+                {"type": "string", "name": "f1"},
+                {"type": "string", "name": "f2", "default": "foo"},
+            ],
+        }
+    )
+)
+schema3 = parse_avro_schema_definition(
+    json.dumps(
+        {
+            "type": "record",
+            "name": "myrecord",
+            "fields": [
+                {"type": "string", "name": "f1"},
+                {"type": "string", "name": "f2"},
+            ],
+        }
+    )
+)
+schema4 = parse_avro_schema_definition(
+    json.dumps(
+        {
+            "type": "record",
+            "name": "myrecord",
+            "fields": [
+                {"type": "string", "name": "f1_new", "aliases": ["f1"]},
+            ],
+        },
+    )
+)
+schema6 = parse_avro_schema_definition(
+    json.dumps(
+        {
+            "type": "record",
+            "name": "myrecord",
+            "fields": [
+                {"type": ["null", "string"], "name": "f1", "doc": "doc of f1"},
+            ],
+        }
+    )
+)
+schema7 = parse_avro_schema_definition(
+    json.dumps(
+        {
+            "type": "record",
+            "name": "myrecord",
+            "fields": [
+                {"type": ["null", "string", "int"], "name": "f1", "doc": "doc of f1"},
+            ],
+        }
+    )
+)
+
+schema8 = parse_avro_schema_definition(
+    # This is invalid JSON
+    '{"type":"record","name":"myrecord","fields":[{"type":"string","name":"f1"},{"type":"string",'
+    '"name":"f2","default":"foo"}]},{"type":"string","name":"f3","default":"bar"}]}'
+)
+# badDefaultNullString = parse_avro_schema_definition(
```

Comment:
I didn't remove these because it indicates we are missing tests with the commented out schemas